src/Main.elm

```diff
@@ -82,172 +140,423 @@ subscriptions model =
 -- VIEW
 view : Model -> Html Msg
 view model =
-  case model.status of
+  case model.loadDataStatus of
     Failure ->
       text "Unable to load Wikipedia Data"
 
     Loading ->
       text "Loading..."
 
     Success wikipediaRecordsArray ->
+
       div[][
+      -- header  
+      div[css[Tw.flex, 
+              Tw.pl_8, 
+              Tw.text_left, 
+              Tw.text_color Tw.custom_pink,
+              Tw.border_b_8,
+              Tw.border_color Tw.black,
+              Tw.bg_radial_gradient,
+              Tw.py_4,
+              Tw.font_serif
+             ]]
+             [h1[][text("NerdSwipe")]],
              
-      div[class "header"][h1[][text("NerdSwipe")]],
-      
-      div [class "main"]
+      -- main container
+      div [css[
+            Tw.flex, 
+            Tw.flex_col, 
+            Tw.justify_between]]
       [ 
-        div[class "gallery"][
-          div[class "alt-wikipedia-container"][
-
-          -- Back Button
-         div[class "previous-button-container"][
-            button [onClick Back, class "button"] 
+      -- gallery container
+      div[css [Tw.flex, 
+               Tw.flex_row, 
+               Tw.justify_between, 
+               Tw.bg_color Tw.black,
+               Tw.border_b_8,
+               Tw.border_color Tw.black,
+               Tw.pr_10,
+               Tw.pl_10,
+          Breakpoints.lg[
+            Tw.pr_4,
+            Tw.pl_4
+          ]]][
+          
+        -- previous wikipedia container
+          div[css[Tw.flex, 
+                  Tw.items_center,
+                  Tw.justify_center]][
+
+        -- previous button
+         div[css [Tw.z_10,
+                  Tw.absolute, 
+                  Tw.transform,
+                  Tw.translate_x_36,Tw.translate_y_0]][
+
+            button [onClick Previous,
```

Comment:
Feels like "button" should be reused.

Reply:
Yes. I will refactor the code to reuse the button and other components/styles.